### DISCUSSION TITLE
Being logged out continuously

### DISCUSSION THREAD

**Discussion by mvhaen** [Budibase/budibase#7085] [Help] [answered]
Anyone else experiencing issues with the Free tier and getting logged out all of a sudden?

**Comment by R2bEEaton** [comment]
Have you been logging in on multiple devices?

I'm not sure if this is exactly the problem here, but I have noticed that any time you log in to Budibase with the same account on another client, it logs you out on the first one. This could also be affected if your browser isn't set to save cookies or if you are using an Incognito or Private Browsing mode.

**Comment by kasgithub8** [comment]
> Anyone else experiencing issues with the Free tier and getting logged out all of a sudden?

Yes, all of our users.

**Comment by melohagan** [comment, marked as answer]
Hey @mvhaen @kasgithub8 

The team has addressed this issue, and the fix should be available by pulling down the latest Budibase version: **v1.2.9**
